pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -836,11 +843,14 @@ void Panels::MemoryCardListPanel_Simple::UiRenameCard(McdSlotItem& card)
 
 	const wxDirName basepath(m_listview->GetMcdProvider().GetMcdPath());
 	wxString newFilename;
-	while (1)
-	{
+	wxString toPrint = Path::ToWxString(card.Filename);
+	while (1){
```

Comment:
Ditto.

Reply:
The clang formatter returns  
`	
wxString toPrint = Path::ToWxString(card.Filename);
while (1) {
`

Reply:
I believe it's supposed to do
```
while(1)
{
```
unless something changed while I was away.